### ISSUE TITLE
analyzer version compatibility issues when used with built_value_generator

### ISSUE THREAD

**Issue by oysterpack** [xsahil03x/super_enum#31]
After upgrading packages,  built_value_generator ^7.0.8 and super_enum_generator ^0.3.0 are conflicting on analyzer version:
```
Because super_enum_generator >=0.2.0+1 depends on analyzer >=0.37.1 <0.39.1 and
built_value_generator >=7.0.8 depends on analyzer ^0.39.3, super_enum_generator >=0.2.0+1 
is incompatible with built_value_generator >=7.0.8.

pub upgrade failed (1; So, because ___ depends on both built_value_generator ^7.0.8 
and super_enum_generator ^0.3.0, version solving failed.)
```

**Comment by xsahil03x** [comment]
Hey @oysterpack , thanks for opening an issue. We will fix it ASAP until then you override the analyzer dependency.

**Comment by xsahil03x** [comment]
Waiting on [#33, #32 ] to get merged first

**Comment by SandroMaglione** [comment]
I have a conflict also with `json_serializable` that it does not even run without the newest version of `analyzer`.

Do you know if there is a quick-fix solution to run `super_enum` with a previous version of `analyzer` while another package with the newest one? How is the situation on #33 and #32?

**Comment by xsahil03x** [comment]
Fixed in #38
